storage/badger/operation/spork.go

```diff
@@ -6,26 +6,14 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 )
 
-// InsertSporkID inserts the spork ID for the present spork. A single database
+// InsertSporkRootBlockID inserts the spork root block ID for the present spork. A single database
 // and protocol state instance spans at most one spork, so this is inserted
 // exactly once, when bootstrapping the state.
-func InsertSporkID(sporkID flow.Identifier) func(*badger.Txn) error {
-	return insert(makePrefix(codeSporkID), sporkID)
+func InsertSporkRootBlockID(blockID flow.Identifier) func(*badger.Txn) error {
```

Comment:
```suggestion
func IndexSporkRootBlock(blockID flow.Identifier) func(*badger.Txn) error {
```

this seems more accurate to me